.gitignore

```diff
@@ -48,3 +50,7 @@ paper/*.aux
 paper/*.log
 paper/*.out
 paper/*.pdf
+
+# tmp directory
+tmp/
```

Comment:
Please remove this, since I suppose it's user specific. There is no reason to add user specific stuff to .gitignore.